src/controllers/trip.controller.js

```diff
@@ -175,6 +176,10 @@ class TripController {
   static async createComment(req, res) {
     const Id = req.params.tripRequestID;
     const data = await commentService.createComment(req, Id, 'trip request');
+    const tripRequest = await tripService.findRequestByID(db.requesttrip, { tripId: Id });
+    const tripRequestData = tripRequest[0].dataValues;
+    const receiver = (req.user.dataValues.id === tripRequestData.managerId) ? tripRequestData.userId : tripRequestData.managerId;
+    await NotificationService.sendNotification('trip_request_comment_event', receiver, 'New Comment', `${req.user.firstName} has posted a new comment`, data.dataValues.id, `${process.env.BASE_URL}/api/v1/notifications`);
```

Comment:
@shemaeric1234 I guess you're sending email notification too? what will happen if the user clicks view message from the inbox?